config.h

```diff
@@ -13,20 +13,22 @@
  */
 
 // General settings
-#define MY_BAUD_RATE 9600
+#define MY_BAUD_RATE 115200
```

Comment:
Hey long time no see, i'm back on NodeManager. Could you instead make a copy in a config.h.sample file, and in gitignore you could add config.h; ? It would be more comfortable for commit / usage
Of course sometime the config.h file will have to be updated by the user...